<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Inserimento Dipendente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(254, 235, 156);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="Inserimento_dipendente">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 164, 144);</string>
     </property>
     <property name="text">
      <string>Inserimento Dipendente</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_torna">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 border-radius: 14px;
Width:28px;
Height:28px;
background-color: rgb(55, 92, 108);
color: rgb(255, 255, 255);
border-style: outset;


}

QPushButton:hover{

background-color: rgb(92, 154, 181);
}
QPushButton:pressed{
border-style: inset;
background-color: rgb(48, 80, 91);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../images/Torna-Indietro-icon.png</normaloff>../../../images/Torna-Indietro-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Home">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
 border-radius: 14px;
Width:28px;
Height:28px;
background-color: rgb(55, 92, 108);
color: rgb(255, 255, 255);
border-style: outset;


}

QPushButton:hover{

background-color: rgb(92, 154, 181);
}
QPushButton:pressed{
border-style: inset;
background-color: rgb(48, 80, 91);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../images/Home-icon.png</normaloff>../../../images/Home-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>17</width>
         <height>17</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>In questa sezione si pu√≤ aggiungere un nuovo dipendente nel sistema inserendo tutti i dati richiesti</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="error">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>10</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(229, 51, 51);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);
alternate-background-color: rgb(255, 85, 0);</string>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Nome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;
	padding: 5px;
	Height:13px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Cognome:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cognome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px;
Height:13px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Sesso:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Sesso">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
	background: white;
	border-radius:10px;
	padding: 5px;
}


QComboBox QAbstractItemView {
	selection-background-color:rgb(55, 92, 108);
	background-color: rgb(255, 255, 255);
	border-radius:5px;
	padding: 5px;

}


QComboBox::down-arrow {
	 image: url(G:/Secondo anno automazione/Ingegneria del softwear/Cinema/GestioneCinema/images/ComboBox-arrow.png) ;
}

QComboBox::drop-down{
	border-radius:10px;
	padding: 5px;
}

QComboBox::drop-down:hover{
	
	background-color: rgb(92, 176, 150);
	border-radius:5px;
	padding: 5px;
}

QComboBox::down-arrow:on {
    top: 1px;
}

</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>M</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Data di nascita:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="Data_di_nascita">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit{
	background-color: rgb(255, 255, 255);
	border-radius:10px;
	padding: 5px;
}

QDateEdit::down-arrow {
 image: url(G:/Secondo anno automazione/Ingegneria del softwear/Cinema/GestioneCinema/images/ComboBox-arrow.png) ;
}


QDateEdit::up-arrow{
		 image: url(G:/Secondo anno automazione/Ingegneria del softwear/Cinema/GestioneCinema/images/ComboBox-up-arrow.png);
}


QDateEdit::up-arrow:on {
	bottom: 1px;
}


QDateEdit::down-button{
	border-radius:10px;
	padding: 5px;

}

QDateEdit::down-button:hover{
	background-color: rgb(92, 164, 144);
	border-radius:6px;
	padding: 6px;
}

QDateEdit::down-button:pressed{
	
	background-color: rgb(255, 255, 255);
	
}

QDateEdit::up-button{
	border-radius:10px;
	padding: 5px;

}

QDateEdit::up-button:hover{
	background-color: rgb(92, 164, 144);
	border-radius:6px;
	padding: 6px;
}

QDateEdit::up-button:pressed{
	
	background-color: rgb(255, 255, 255);
	
}




</string>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Luogo di nascita:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Luogo_di_nascita">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px;
Height:13px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Mansione</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Mansione">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
	background-color: rgb(255, 255, 255);
	border-radius:10px;
	padding: 5px;
}

QComboBox QAbstractItemView {
	selection-background-color:rgb(55, 92, 108);
	background-color: rgb(255, 255, 255);
	border-radius:5px;
	padding: 5px;

}


QComboBox::down-arrow {
	 image: url(G:/Secondo anno automazione/Ingegneria del softwear/Cinema/GestioneCinema/images/ComboBox-arrow.png) ;
}

QComboBox::drop-down{
	border-radius:10px;
	padding: 5px;
}

QComboBox::drop-down:hover{
	
	background-color: rgb(92, 176, 150);
	border-radius:5px;
	padding: 5px;
}

QComboBox::down-arrow:on {
	top: 1px;
}


</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Addetto Bar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Addetto Biglietteria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Addetto Controllo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Addetto Pulizia</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Codice fiscale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Codice_fiscale">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px;
Height:13px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Stipendio:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Stipendio">
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px;
Height:13px;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(92, 164, 144);</string>
     </property>
     <property name="text">
      <string>Commento:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QTextEdit" name="Commento">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{
background-color: rgb(255, 255, 255);
border-radius: 25px;
padding: 5px;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_inserisci_dip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 12px;
padding: 5px;
background-color: rgb(55, 92, 108);
color: rgb(255, 255, 255);
border-style: outset;
Height:20px;
Width:50px;
}
QPushButton:hover{
background-color: rgb(92, 154, 181);
}
QPushButton:pressed{
border-style: inset;
background-color: rgb(48, 80, 91);
}</string>
       </property>
       <property name="text">
        <string>Inserisci</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../images/Inserisci-Dipendente-icon.png</normaloff>../../../images/Inserisci-Dipendente-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
